docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md

```diff
@@ -76,7 +78,7 @@ In this case, an option to automatically add or enable the nonexistent fields to
 Remember to copy and secure the subscription URL. If you closed the window and could not copy the URL, don't worry. Click on the **Edit** button under the list of sources and copy the URL.
 :::
 
-11. Optional: You can create any **Processing Rules** that you want for the AWS Source. Refer to the [Create any Processing Rules](/docs/send-data/collection/processing-rules/create-processing-rule).
+11. Optional: You can create any **Processing Rules** that you want for the AWS Source. Refer to the [Create any Processing Rules](/docs/send-data/collection/processing-rules/create-processing-rule.md).
 
 ### Step 4: Update AWS Security Data Lake Subscriber
```

Comment:
Fix broken link in step 3:

3. Add the subscription URL copied from the AWS Security Data Lake source setup in [Create AWS Security Data Lake Source](#step-3-create-aws-security-data-lake-source-using-sumo-logic-environment) section.